### ISSUE TITLE
Upload XML client config does not work in MC 4.0

### ISSUE THREAD

**Issue by puzpuzpuz** [hazelcast/management-center-docker#23]
Steps to reproduce:
1. Run `docker run -p 8080:8080 hazelcast/management-center:4.0`
2. Configure security provider and then go to Add Cluster Config
3. Upload the following XML client config:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<hazelcast-client xmlns="http://www.hazelcast.com/schema/client-config"
                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                  xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
                  http://www.hazelcast.com/schema/client-config/hazelcast-client-config-4.0.xsd">
    <cluster-name>Cluster B</cluster-name>

    <network>
        <cluster-members>
            <address>127.0.0.1:5701</address>
            <address>127.0.0.2:5702</address>
        </cluster-members>
    </network>
</hazelcast-client>
```

Expected result:
Config is successfully created (just like it happens with `java -jar` scenario)

Actual result:
An error happens with the following log output:
```
2020-02-18 06:54:21 [qtp1357563986-16] WARN  org.eclipse.jetty.server.HttpChannel - /api/clusters
org.springframework.web.util.NestedServletException: Request processing failed; nested exception is com.hazelcast.core.HazelcastException: org.xml.sax.SAXParseException; schema_reference.4: Failed to read schema document 'null', because 1) could not find the document; 2) the document could not be read; 3) the root element of the document is not <xsd:schema>.
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014)
        ...
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: com.hazelcast.core.HazelcastException: org.xml.sax.SAXParseException; schema_reference.4: Failed to read schema document 'null', because 1) could not find the document; 2) the document could not be read; 3) the root element of the document is not <xsd:schema>.
	at com.hazelcast.internal.util.ExceptionUtil.lambda$static$0(ExceptionUtil.java:39)
	at com.hazelcast.internal.util.ExceptionUtil.peel(ExceptionUtil.java:122)
	at com.hazelcast.internal.util.ExceptionUtil.peel(ExceptionUtil.java:67)
	at com.hazelcast.internal.util.ExceptionUtil.rethrow(ExceptionUtil.java:127)
	at com.hazelcast.client.config.XmlClientConfigBuilder.build(XmlClientConfigBuilder.java:159)
	at com.hazelcast.client.config.XmlClientConfigBuilder.build(XmlClientConfigBuilder.java:150)
	at com.hazelcast.client.config.XmlClientConfigBuilder.build(XmlClientConfigBuilder.java:145)
	at com.hazelcast.webmonitor.repositories.sql.ClusterRepository.parseClientConfigFile(ClusterRepository.java:176)
	at com.hazelcast.webmonitor.repositories.sql.ClusterRepository.add(ClusterRepository.java:47)
	at com.hazelcast.webmonitor.service.ClusterManager.add(ClusterManager.java:90)
	at com.hazelcast.webmonitor.controller.api.ClusterController.addCluster(ClusterController.java:126)
	...
	... 77 common frames omitted
```

**Comment by puzpuzpuz** [comment]
This issue is not related with Docker image and should be fixed in MC 4.0.1 release.

**Comment by yuraku** [comment]
Verified in 4.0.1
